packages/angular_devkit/build_angular/src/builders/browser/schema.json

```
@@ -256,7 +256,7 @@
     },
     "vendorChunk": {
       "type": "boolean",
-      "description": "Generate a seperate bundle containing only vendor libraries. This option should only used for development.",
+      "description": "Generate a seperate bundle containing only vendor libraries. This option should only used for development to reduce the incremental compilation time.",
```

```suggestion
      "description": "Generate a seperate bundle containing only vendor libraries. This option should only be used for development to reduce the incremental compilation time.",
```
same for other places as well